core/templates/dev/head/pages/skill-editor-page/editor-tab/skill-misconceptions-editor/misconception-editor.directive.html

```diff
@@ -31,9 +31,9 @@
     <div ng-if="isEditable() && nameEditorIsOpen">
       <div class="oppia-rule-details-header">
         <strong>Misconception Name</strong>
-        <input class="form-control" type="text" ng-model="container.misconceptionName" maxlength="30">
-        <span ng-if="container.misconceptionName.length === 30" class="form-text" style="color: red; font-size: smaller">
-          <em>Please use at most 30 characters.</em>
+        <input class="form-control" type="text" ng-model="container.misconceptionName" maxlength="50">
```

Comment:
I thought we were planning to go for 100? At least that's what it says in the meeting notes.

We should also probably make this into a global constant and use it everywhere, to prevent skew like last time.

Reply:
Done